packages/wonder-blocks-form/components/choice-internal.js

```diff
@@ -53,14 +53,28 @@ type Props = {|
     variant: "radio" | "checkbox",
 |};
 
+const defaultHandlers = {
+    onClick: () => void 0,
+    onMouseEnter: () => void 0,
+    onMouseLeave: () => void 0,
+    onMouseDown: () => void 0,
+    onMouseUp: () => void 0,
+    onTouchStart: () => void 0,
+    onTouchEnd: () => void 0,
+    onTouchCancel: () => void 0,
+    onKeyDown: () => void 0,
+    onKeyUp: () => void 0,
+    onFocus: () => void 0,
+    onBlur: () => void 0,
+};
+
 /**
  * This is a labeled 🔘 or ☑️ item. This is an internal component that's wrapped
  * by ChoiceField or Choice. Choice should be used in a CheckboxGroup or in a
  * RadioGroup. ChoiceField is the variant used outside of such a group. The two
  * are different to allow for more explicit flow typing. Choice has many of its
  * props auto-populated, but ChoiceField does not.
- */
-export default class ChoiceInternal extends React.Component<Props> {
+ */ export default class ChoiceInternal extends React.Component<Props> {
```

Comment:
This is the weird spacing issue that @wishdasher was seeing with Prettier, would be good to fix this! Maybe we need to upgrade Prettier?

Reply:
Updating prettier does not resolve this issue.

Reply:
It seems like this would be pretty easy to fix, just add a rule that states code should not appear on a line after a block comment.

Reply:
I'm going to leave this as is otherwise we'll end up with this change the next time someone edits this file.